config/eslint.node.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
```

Comment:
We should use semi-colons `;` to stay aligned with other related projects.

Reply:
+1

Reply:
Our ESLint configuration is currently based on [StandardJS](https://standardjs.com/) which eliminates the need for semicolons. Technically, semicolons in JavaScript are unnecessary [except a few edge cases](https://standardjs.com/rules.html#semicolons) due to the ASI (automatic semicolon insertion) mechanism being part of ECMAScript language spec ([link](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-rules-of-automatic-semicolon-insertion)).

We already have a good experience using StandardJS as the code style baseline in oVirt projects like [ovirt-web-ui](https://github.com/oVirt/ovirt-web-ui) and [ovirt-engine-ui-extensions](https://gerrit.ovirt.org/gitweb?p=ovirt-engine-ui-extensions.git;a=tree).

When it comes to contributing our components to other projects (like `patternfly-react`), we can use their [Prettier](https://prettier.io/) config associated with the given project. Typically, this involves copying the component into such project and using Prettier-aware editor to save & automatically format the code.

To summarize, I don't think that not using semicolons creates that big of a difference between us and other projects like `patternfly-react`. Also, check out [`eslint-plugin-patternfly-react` recommended config](https://github.com/patternfly/patternfly-react/blob/master/packages/eslint-plugin-patternfly-react/lib/config/recommended.js) - it's actually based on StandardJS, but also applies AirBnb on top of it (this one re-adds the semicolons).

Reply:
+1 for consistent lint configs. I'm not sure what warrants the new config here, but would like to stay consistent w/ patternfly and [console](https://github.com/openshift/console) where possible. My guess is we will eventually introduce a tslint config in pf4 react, but for now, eslint-plugin-patternfly-react is the closest config in PF if you'd like to share any configs introduced there. Not overly pressing, but consistency is nice as we share components across projects.

Reply:
Having independent project, I would not like to follow all of the pf-react linting rules.
But since this is not our case, switching of context in our heads will cause much more stress than getting used to it.